### PR TITLE
Adding test cases OCS-1009 and OCS-1018

### PR DESCRIPTION
**Pull request by jilju**
OCS-1009 - RBD: Delete csi-rbdplugin while deleting app pods along with their mount points
OCS-1018 - CEPHFS: Delete csi-cephfsplugin while deleting app pods along with their mount points

Signed-off-by: Jilju Joy <jijoy@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/resources/pod.py [hunk 1]
```diff
@@ -850,3 +850,37 @@ def get_used_space_on_mount_point(pod_obj):
     mount_point = mount_point.split()
     used_percentage = mount_point[mount_point.index(constants.MOUNT_POINT) - 1]
     return used_percentage
+
+
+def get_cephfsplugin_pods(cephfsplugin_label=constants.CSI_CEPHFSPLUGIN_LABEL, namespace=None):
```
Comment (RazTamir) [inline ocs_ci/ocs/resources/pod.py]: please combine both functions and distinguish with passing interface parameter

File: tests/disruption_helpers.py [hunk 2]
```diff
@@ -22,17 +22,27 @@ def set_resource(self, resource):
...
+            self.type = 'rook-ceph'
         if self.resource == 'mon':
             self.resource_obj = pod.get_mon_pods()
+            self.type = 'rook-ceph'
         if self.resource == 'osd':
             self.resource_obj = pod.get_osd_pods()
+            self.type = 'rook-ceph'
         if self.resource == 'mds':
             self.resource_obj = pod.get_mds_pods()
+            self.type = 'rook-ceph'
```
Comment (ebenahar) [inline tests/disruption_helpers.py]: @jilju 
Great!! In fact, I was wondering if we're going to enhance this class

File: ocs_ci/ocs/resources/pod.py [hunk 3]
```diff
@@ -850,3 +850,37 @@ def get_used_space_on_mount_point(pod_obj):
...
+    Returns:
+        list : csi-cephfsplugin pod objects
+    """
+    namespace = namespace or config.ENV_DATA['cluster_namespace']
+    cephfsplugins = get_pods_having_label(cephfsplugin_label, namespace)
+    cephfsplugin_pods = [Pod(**cephfsplugin) for cephfsplugin in cephfsplugins]
+    return cephfsplugin_pods
+
+
+def get_rbdplugin_pods(rbdplugin_label=constants.CSI_RBDPLUGIN_LABEL, namespace=None):
```
Comment (vasukulkarni) [inline ocs_ci/ocs/resources/pod.py]: can you move this function into utils, dont think they are class methods.

**Comment by jilju** [comment]
> PR validation [Job FAILED](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/221//testReport) (installation failed, tests not executed).

This will be fixed in https://github.com/red-hat-storage/ocs-ci/pull/762

**Pull request merged**